docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.md

```diff
@@ -236,9 +237,36 @@ scaled = zne.scaling.scale_parameters(circuit2, scale_factor=2.0, base_variance=
 print("\nScaled circuit:", scaled, sep="\n")
 ```
 
+**Codeblock 13**
+
++++
+
+> Note: The paper shows pseudocode; here we show an example.
+
+```{code-cell} ipython3
+from mitiq.zne.scaling import compute_parameter_variance
+
+# Estimate base level of parameter noise
+base_variance = compute_parameter_variance(executor, cirq.X, cirq.LineQubit(0))
+print("Estimation of parameter noise variance:", base_variance)
+
+def scale_param_noise(circuit, scale_factor):
+    return zne.scaling.scale_parameters(circuit, scale_factor, base_variance=0.009)
+
+zne_value = zne.execute_with_zne(
+    circuit,
+    executor,
+    scale_noise=scale_param_noise,
+    num_to_average=10,
+)
+# We are applying parameter-noise mitigation with an executor which is instead subject to depolarizing noise.
+# Therefore, we don't expect good performances.
+print("ZNE value via parameter noise scaling:", zne_value)
+```
+
 +++ {"id": "ye9I9WLNrGxU"}
 
-**Codeblock 13**
+**Codeblock 14**
```

Comment:
1. Nice catch.
2. Sucks this has to be done manually.

Reply:
Hopefully, after the paper is published, they won't be changed for all eternity :-D